sdk/eventhub/Azure.Messaging.EventHubs.Shared/tests/BlobCheckpointStore/BlobsCheckpointStoreInternalTests.cs

```diff
@@ -413,6 +413,11 @@ public async Task GetCheckpointUsesSequenceNumberAsTheStartingPositionWhenNoOffs
 
             Assert.That(checkpoint, Is.Not.Null, "A checkpoint should have been returned.");
             Assert.That(checkpoint.StartingPosition, Is.EqualTo(expectedStartingPosition));
+
+            Assert.That(checkpoint, Is.InstanceOf<BlobCheckpointStoreInternal.BlobStorageCheckpoint>());
+            var blobCheckpoint = (BlobCheckpointStoreInternal.BlobStorageCheckpoint)checkpoint;
+            Assert.IsNull(blobCheckpoint.Offset);
```

Comment:
Could we please stick to the `Assert.That` format and include a failure message since there are multiple asserts in this test?